### PR TITLE
Tableau: test scripts

### PR DESCRIPTION
**Pull request by dsager**
This is some test code i wrote when evaluating the tableau trial...

### CODE REVIEW

File: projects/tableau-playground/test_publish.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from tableausdk.Server import *
+
+try:
+    ServerAPI.initialize()
+    serverConnection = ServerConnection()
+    serverConnection.connect('https://10ay.online.tableau.com',
+                             os.environ['TAB_SDK_USER'],
+                             os.environ['TAB_SDK_PASSWORD'],
+                             'devex')
+    serverConnection.publishExtract('test.tde',
```
Comment (nicolas-fricke) [inline projects/tableau-playground/test_publish.py]: `camelCased` functions in Tableau's Python SDK? Real Pythonists would send whoever wrote that to the coder's dungeon for that :dragon_face:

Comment (dsager) [inline projects/tableau-playground/test_publish.py]: maybe to keep it consistent with their Java SDK? this is just some testing stuff which i didn't want to throw away. so don't pay too much attention reviewing ;)

**Comment by nicolas-fricke** [comment]
:+1: Since it's only for testing.. otherwise `import *` is somewhat discouraged in the Python world since it easily overloads things in the namespace and it makes it quite hard to find out where certain functionality comes from.

File: projects/tableau-playground/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- `TAB_SDK_PASSWORD`
+- `TAB_SDK_USER`
+- `TAB_SDK_LOGDIR`
+
+## Install Tableau SDK
+
+    wget http://downloads.tableausoftware.com/tssoftware/Tableau-SDK-Python-Linux-64Bit.tar.gz
+    tar -xzf Tableau-SDK-Python-Linux-64Bit.tar.gz
+    cd TableauSDK*
+    sudo python setup.py install
```
Comment (ifosch) [inline projects/tableau-playground/README.md]: I would recommend trying conda, from [Miniconda](http://conda.pydata.org/miniconda.html)

File: projects/tableau-playground/test_publish.py (hunk 1 above)
Comment (ifosch) [inline projects/tableau-playground/test_publish.py]: That's right, though other libraries in the stdlib are also doing that, to name one, unittest.

**Comment by ifosch** [comment]
+1 Very nice code, I took a look on Tableau not very long ago... willing to see it in action.

**Pull request merged**